### DISCUSSION TITLE
Some questions on auto-refresh

### DISCUSSION THREAD

**Discussion by kkalata** [FreshRSS/FreshRSS#6598] [Q&A] [answered]
1. How to change auto-refresh frequency on running Docker container?
2. Is there a way for the feed to be auto-refreshed at certain time, eg. every full hour or on Mondays at 12 PM?
3. Is "do not automatically refresh more often than" option related to the time it is set? For example, if I set the feed not to be auto-refreshed more often than one week on Sunday at 8 PM, does it mean that it will be auto-refreshed every Sunday at 8 PM?
4. Is counter of the option mentioned above reset on manual refresh?

**Comment by Alkarex** [comment, marked as answer]
1. See `CRON_MIN` https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md
2. Not at the moment
3. The minimal next refresh is calculated from the last refresh
4. Yes
